scripts/prepare/bundle.ts

```diff
@@ -32,40 +32,40 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
     await fs.emptyDir(join(process.cwd(), 'dist'));
   }
 
-  if (!optimized) {
-    await Promise.all(
-      entries.map(async (file: string) => {
-        console.log(`skipping generating types for ${file}`);
-        const { name: entryName, dir } = path.parse(file);
-
-        const pathName = join(process.cwd(), dir.replace('./src', 'dist'), `${entryName}.d.ts`);
-        const srcName = join(process.cwd(), file);
-
-        const rel = relative(dirname(pathName), dirname(srcName))
-          .split(path.sep)
-          .join(path.posix.sep);
-
-        await fs.ensureFile(pathName);
-        await fs.writeFile(
-          pathName,
-          dedent`
-          // devmode
-          export * from '${rel}/${entryName}';
-        `
-        );
-      })
-    );
-  }
-
   const tsConfigPath = join(cwd, 'tsconfig.json');
   const tsConfigExists = await fs.pathExists(tsConfigPath);
+
   await Promise.all([
+    // SHIM DTS FILES (only for development)
+    ...(optimized
+      ? []
+      : entries.map(async (file: string) => {
+          const { name: entryName, dir } = path.parse(file);
+
+          const pathName = join(process.cwd(), dir.replace('./src', 'dist'), `${entryName}.d.ts`);
+          const srcName = join(process.cwd(), file);
+
+          const rel = relative(dirname(pathName), dirname(srcName))
+            .split(path.sep)
+            .join(path.posix.sep);
+
+          await fs.ensureFile(pathName);
+          await fs.writeFile(
+            pathName,
+            dedent`
+              // dev-mode
+              export * from '${rel}/${entryName}';
+            `
+          );
+        })),
+
+    // BROWSER EMS
     build({
+      silent: true,
       entry: entries.map((e: string) => slash(join(cwd, e))),
       watch,
       ...(tsConfigExists ? { tsconfig: tsConfigPath } : {}),
       outDir: join(process.cwd(), 'dist'),
-      // sourcemap: optimized,
       format: ['esm'],
       target: 'chrome100',
```

Comment:
@JReinhold you mentioned that we might want to add a FireFix version in here?
What do you think that version should be?

Reply:
Based on [browserslist defaults](https://browsersl.ist/#q=defaults) and [Can I Use usage table](https://caniuse.com/usage-table) I'd suggest this:
```suggestion
      target: ['chrome105', 'safari15', 'firefox105', 'ios15'],
```
Safari 15 and iOS 15 might be redundant, I honestly don't know.

Reply:
FWIW, our migration guide indicates chrome 100+, if we change this, we would probably need to update that doc as well.